src/extensions/old/curation/CurationSortingView/AllUnitsTab.tsx

```diff
@@ -22,7 +22,9 @@ const AllUnitsTab: FunctionComponent<SortingViewProps> = (props) => {
                             <sv.component
                                 {...sv.props || {}}
                                 sorting={props.sorting}
+                                sortingInfo={props.sortingInfo}
```

Comment:
For convenience, we now pass sortingInfo and recordingInfo explicitly into SortingView plugins. This data is no longer attached to sorting and recording.